components/TARGET_PSA/services/storage/its/psa_prot_internal_storage.h

```diff
@@ -120,8 +120,8 @@ psa_status_t psa_its_set(psa_storage_uid_t uid,
  *                                           is invalid. For example is `NULL` or references memory the caller cannot access
  */
 psa_status_t psa_its_get(psa_storage_uid_t uid,
-                         uint32_t data_offset,
-                         uint32_t data_length,
+                         size_t data_offset,
+                         size_t data_length,
                          void *p_data);
```

Comment:
Per spec, we should have a `size_t *p_data_length` parameter here (for `psa_its_get()`)

Reply:
Will change.